src/main/kotlin/viewModel/graph/UndirectedViewModel.kt

```diff
@@ -48,8 +52,25 @@ class UndirectedViewModel(
         set(value) {
             _ranked.value = value
             ranks = PageRank(graph).computePageRank(3)
+            println("хуй")
             updateSizes()
         }
+    
+    var bridgeFinded
+        get() = _bridgeFinded.value
+        set(value) {
+            _bridgeFinded.value = value
+            bridges = FindBridges(graph).findBridges()
+            bridges.forEach {
+                BridgesWthColor.add(it to Color.Red)
+            }
+            if (bridgeFinded) {
+                changeEdgesColor(BridgesWthColor)
+            }
+            else{
+                resetEdgesColorToDefault()
+            }
```

Comment:
Включи в настройках "Format on save", либо почаще нажимай Ctrl + Shiift + L (форматирование)

```suggestion
            _bridgeFinded.value = value

            bridges = FindBridges(graph).findBridges()
            bridges.forEach {
                BridgesWthColor.add(it to Color.Red)
            }
            
            if (bridgeFinded) {
                changeEdgesColor(BridgesWthColor)
            } else {
                resetEdgesColorToDefault()
            }
```